crates/extricrate/src/lib.rs

```diff
@@ -621,6 +631,38 @@ pub mod dependencies {
                 }
             );
         }
+
+        #[test]
+        fn aliases_resolved() {
+            let src = r#"
+                use foo::bar as baz;
+                use foo::Bar as Baz;
+            "#;
+            let file = syn::parse_file(src).unwrap();
+            let mut visitor = Visitor::default();
+            visitor.visit_file(&file);
+
+            assert_eq!(visitor.use_statements.len(), 2);
+
+            assert_eq!(
+                visitor.use_statements[0].target_modules[0],
+                ModuleName("foo::bar".into())
+            );
+            assert_eq!(
+                visitor.use_statements[0].statement.items,
+                vec![NormalizedUseStatement {
+                    module_name: "foo".into(),
+                    statement_type: UseStatementType::Alias("bar".to_string(), "baz".to_string())
```

Comment:
Following our convention, shouldn't this be
```suggestion
                    module_name: "foo::bar".into(),
                    statement_type: UseStatementType::Alias("self".to_string(), "baz".to_string())
```

Reply:
ok, I've reviewed how we get module names in different places, I think [this commit](https://github.com/nitnelave/extricrate/pull/2/commits/9c6215e695a8c41cce467fce4b81a77968c9a851) should standardise them all. I think `use_statements` should work the same as `target_modules`, let me know if you disagree tho 😄